src/main/java/TaskManager.java

```diff
@@ -0,0 +1,83 @@
+public class TaskManager {
+    public static final int MAX_TASKS = 100;
+    public static final int TODO_TASK_NAME_BEGIN_INDEX = 5;
+    public static final int DEADLINE_TASK_NAME_BEGIN_INDEX = 9;
+    public static final int SKIP_BY_IN_DEADLINE = 4;
+    public static final int EVENT_TASK_NAME_BEGIN_INDEX = 6;
+    public static final int SKIP_FROM_IN_EVENT = 6;
+    public static final int SKIP_TO_IN_EVENT = 4;
+    private int currIndex;
+    private Task[] tasks;
+
+    public TaskManager() {
+        this.currIndex = 0;
+        this.tasks = new Task[MAX_TASKS];
+    }
+
+    public void addTask(String taskToAdd) {
+        if (taskToAdd.contains("todo")) {
+            processToDo(taskToAdd);
+        } else if (taskToAdd.contains("deadline")) {
+            processDeadline(taskToAdd);
+        } else if (taskToAdd.contains("event")) {
+            processEvent(taskToAdd);
+        } else {
+            System.out.println("ERROR: Invalid input");
+            return;
+        }
```

Comment:
Can I understand why the contains method is used to check for the type of command? Consider the input "deadline todo homework".  Will this task be added as a deadline task or todo task? Was the code intended to work such that the todo tasks has a higher precedence over deadline tasks?

Reply:
I neglected to consider the cases where (1) there are more than one keyword in the input and (2) the keyword is not the first word in the input. Changed the if conditions to check for these